packages/vats/tools/fake-bridge.js

```diff
@@ -0,0 +1,80 @@
+import { Fail } from '@agoric/assert';
+import assert from 'node:assert/strict';
+
+/** @import {ScopedBridgeManager} from '../src/types.js'; */
+
+/**
+ * @param {import('@agoric/zone').Zone} zone
+ * @param {(obj) => void} onToBridge
+ * @param {(handler, obj) => Promise<void>} onFromBridge
+ * @returns {ScopedBridgeManager<'dibc'>}
+ */
+export const makeFakeIbcBridge = (zone, onToBridge, onFromBridge) => {
+  let hndlr;
+  return zone.exo('Fake IBC Bridge Manager', undefined, {
+    getBridgeId: () => 'dibc',
+    toBridge: async obj => {
+      onToBridge(obj);
+      const { method, type, ...params } = obj;
+      assert.equal(type, 'IBC_METHOD');
+      if (method === 'sendPacket') {
+        const { packet } = params;
+        return { ...packet, sequence: '39' };
+      }
+      return undefined;
+    },
+    fromBridge: async obj => {
+      if (!hndlr) throw Error('no handler!');
+      await onFromBridge(hndlr, obj);
+    },
+    initHandler: h => {
+      if (hndlr) throw Error('already init');
+      hndlr = h;
+    },
+    setHandler: h => {
+      if (!hndlr) throw Error('must init first');
+      hndlr = h;
+    },
+  });
+};
+
+/**
+ * @param {import('@agoric/zone').Zone} zone
+ * @param {(obj) => void} [onToBridge]
+ * @param {(handler, obj) => ERef<void>} [onFromBridge]
+ * @returns {ScopedBridgeManager<'vlocalchain'>}
+ */
+export const makeFakeLocalchainBridge = (
+  zone,
+  onToBridge = () => {},
+  onFromBridge = () => {},
+) => {
+  let hndlr;
+  return zone.exo('Fake Localchain Bridge Manager', undefined, {
+    getBridgeId: () => 'vlocalchain',
+    toBridge: async obj => {
+      onToBridge(obj);
```

Comment:
Is this the facility for providing a fixture? Seems if we did a ~ `if(typeof onToBridge === 'function') return onBridge(obj);` on L65 (and L24 above) we can supply custom handlers?

Reply:
I didn't get far into solving that. I figure we need those fixtures mostly in RunUtils style tests, and those need synchronous `toBridge` because of where the stand, so this module fake bridge won't work there.